### ISSUE TITLE
Killing ray actors does not work

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/python-gatenlp#161]
Complains about the wrong type of handle or actor. Investigate.

**Comment by johann-petrak** [comment]
```
Traceback (most recent call last):
  File "/home/johann/work-git/python-gatenlp/gatenlp/processing/runners.py", line 308, in siginthandler
    ray.kill(actor)
  File "/home/johann/software/anaconda3/envs/fcdo/lib/python3.8/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/johann/software/anaconda3/envs/fcdo/lib/python3.8/site-packages/ray/worker.py", line 2002, in kill
    raise ValueError("ray.kill() only supported for actors. "
ValueError: ray.kill() only supported for actors. Got: <class 'ray._raylet.ObjectRef'>.
Exception ignored in: 'ray._raylet.check_signals'
Traceback (most recent call last):
  File "/home/johann/work-git/python-gatenlp/gatenlp/processing/runners.py", line 308, in siginthandler
    ray.kill(actor)
  File "/home/johann/software/anaconda3/envs/fcdo/lib/python3.8/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/johann/software/anaconda3/envs/fcdo/lib/python3.8/site-packages/ray/worker.py", line 2002, in kill
    raise ValueError("ray.kill() only supported for actors. "
ValueError: ray.kill() only supported for actors. Got: <class 'ray._raylet.ObjectRef'>.
```

**Comment by johann-petrak** [comment]
Fixed as of aea85e8ffd99e2263ea763a90966d9ab874fc787
